bookmarks/account/views.py

```diff
@@ -70,3 +73,18 @@ def edit(request):
         "account/edit.html",
         {"user_form": user_form, "profile_form": profile_form},
     )
+
+
+@ajax_required
+@require_POST
+@login_required
+def user_follow(request):
+    user_id = request.POST.get("id")
+    action = request.POST.get("action")
+    from_user = request.user
```

Comment:
Replace inline temp refactoring. No point in having them in a temp variable since they are used only once. This takes up an unnecessary attention span and affects readability